### PR TITLE
Check in 30630 display in order

### PR DESCRIPTION
**Pull request by mdewey**
## Description
- Added a sorter to the incoming appointments to guarantee that the appointments are in the right order
- updated loading messages

## Original issue(s)
https://github.com/department-of-veterans-affairs/va.gov-team/issues/30630

## Testing done
- added unit 
- added e2e

### CODE REVIEW

File: src/applications/check-in/pages/CheckIn/index.jsx [hunk 1]
```diff
@@ -79,7 +79,7 @@ const CheckIn = props => {
   );
 
   if (isLoadingData) {
-    return <LoadingIndicator message={'Loading appointment details'} />;
+    return <LoadingIndicator message={'Loading you appointments for today'} />;
```
Comment (acrollet) [inline src/applications/check-in/pages/CheckIn/index.jsx:82]:
```suggestion
    return <LoadingIndicator message={'Loading your appointments for today'} />;
```

File: src/applications/check-in/utils/appointment/index.js [hunk 2]
```diff
@@ -7,4 +7,16 @@ const hasMoreAppointmentsToCheckInto = (appointments, currentAppointment) => {
       .filter(f => f.status === STATUSES.ELIGIBLE).length > 0
   );
 };
-export { hasMoreAppointmentsToCheckInto };
+
+const sortAppointmentsByStartTime = appointments => {
```
[the remark this answers is not in the document]
Comment (mdewey) [reply src/applications/check-in/utils/appointment/index.js:11]: I completely agree! I think a spike that leads to a best practice/standards would be awesome. Do you want to spike that out?

[the remark this answers is not in the document]
Comment (acrollet) [reply src/applications/check-in/utils/appointment/index.js:11]: @mdewey will do!

**Review by acrollet — APPROVED** [review]
nice additions to the tests!

File: src/applications/check-in/utils/appointment/index.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (acrollet) [reply src/applications/check-in/utils/appointment/index.js:11]: https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/30779

**Pull request merged**